test/evergreen.yml

```diff
@@ -429,7 +430,7 @@ functions:
           set -o errexit
           set -o verbose
           ${PREPARE_TEST_ENV}
-          ../../../tools/run_parallel.sh 'nice ./recovery-test.sh "${data_validation_stress_test_args} ${run_test_checkpoint_args}" WT_TEST.$t test_checkpoint' 120
+          ../../../tools/run_parallel.sh 'nice ./recovery-test.sh "${data_validation_stress_test_args} ${run_test_checkpoint_args}" WT_TEST.$t test_checkpoint' 20
```

Comment:
What does this do?

Reply:
This test runs for 120 iterations, but with TSan being 10 times slower we hit time outs. 
I found that 20 iterations is short enough to complete in time.

Reply:
This is a substantial decrease in testing across the board but tbh probably a welcome one.

Reply:
This was just a patch to make things green. I'm not proposing it as a long term solution.
I'll think on how we handle this cleanly

Reply:
Updated in [d4faf47](https://github.com/wiredtiger/wiredtiger/pull/10812/commits/d4faf4799bd21e890f2e1c94debc2181d070f870) to special case for the No Throw variant